python/tvm/relay/frontend/tflite.py

```diff
@@ -2280,6 +2276,20 @@ def get_expr(self, input_tensor_idx):
     def has_expr(self, input_tensor_idx):
         return self.exp_tab.has_expr(get_tensor_name(self.subgraph, input_tensor_idx))
 
+    def get_tensor_expr(self, tensor):
```

Comment:
@FrozenGene, @siju-samuel, Corrected the func name here also based on the comment on https://github.com/apache/incubator-tvm/pull/5486